projects/packages/search/src/class-search-products.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Class to fetch Search product pricing
+ *
+ * @package    automattic/jetpack-search
+ */
+
+namespace Automattic\Jetpack\Search;
+
+/**
+ * Class to fetch Search product pricing
+ *
+ * @package Automattic\Jetpack\Search
+ */
+class Search_Products {
```

Comment:
Not sure we have a Jetpack convention on this, but have we used singular naming for other classes (so `Search_Product`)?

Reply:
Good question, renamed to `Product` 😄 a97f135